assets/locales/en/main.json

```diff
@@ -59,9 +59,44 @@
     "having-trouble": "Having trouble…",
     "no-street": "No street selected.",
     "trouble-loading": "We’re having trouble loading Streetmix.",
+    "error-code": "(Error {{code}}.)",
     "slow-connection-1": "Streetmix wasn’t able to connect to the Internet in awhile now.",
     "slow-connection-2": "You might want to reload the page and try again. Please note you might lose the latest change to the street. Sorry!",
     "no-connection": "Streetmix is having trouble connecting to the Internet."
+  },
+	"errors": {
+		"buttons": {
+			"home-button": "Go to the homepage",
+			"sign-in-button": "Sign in again",
+			"reload-button": "Reload the page",
+			"try-again-button": "Try again",
+			"general-help-buttons": "<a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>. <br /> <button id = 'error-home'onClick = {goHome} > Go to the homepage < /button>"
+		},
+		"page-not-found-title": "Page not found.",
+		"page-not-found-description": "Oh, boy. There is no page with this address!<br />{homeButton}",
+		"street-not-found-title": "Street not found.",
+		"street-not-found-description": "Oh, boy. There is no street with this link!<br />{homeButton}",
+		"street-not-found-but-user-description": "There is no street with this link! But you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"street-deleted-title": "This street has been deleted.",
+		"street-deleted-description": "There is no longer a street with this link, but you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"sign-out-title": "You are now signed out.",
+		"reload-sign-out-title": "You signed out in another window.",
+		"gallery-street-failure-description": "We’re having trouble loading this street.",
+		"reload-sign-out-description-401": "Please reload this page before continuing.<br />(Error RM2.)<br /><button id='error-clear-sign-in-reload' onClick={goReloadClearSignIn}>Reload the page</button>",
```

Comment:
Remove everything after, and including `<br />` (the first one). Notice that we can handle the middle string with the `error-code` string and the latter part within React and the `error.button.reload` string.

Reply:
Ahhh, this is great for me to learn. Now I'm starting to understand things a bit more.

Reply:
@louh This question may be too much for now, but how are these strings brought to the actual  UI? Where in the code does this happen. Is someone having to look at this translation file and reference all of these different keys we are creating?